specification/network/resource-manager/Microsoft.Network/preview/2023-08-01-preview/examples/NspLoggingConfigurationDelete.json

```diff
@@ -0,0 +1,18 @@
+{
+  "title": "Delete NSP logging configuration",
+  "parameters": {
+    "api-version": "2023-08-01-preview",
+    "subscriptionId": "subId",
+    "resourceGroupName": "rg1",
+    "networkSecurityPerimeterName": "nsp1"
+  },
+  "responses": {
+    "200": {},
+    "202": {
```

Comment:
Delete logging config is not async API, hence there won't be 202  response for delete.